tests/tabular/checks/model_evaluation/train_test_prediction_drift_test.py

```diff
@@ -124,3 +137,57 @@ def test_drift_max_drift_score_condition_pass_threshold(drifted_data_and_model):
         details='Found model prediction PSI drift score of 0.79',
         name='categorical drift score < 1 and numerical drift score < 1'
     ))
+
+
+def test_multiclass_proba(iris_split_dataset_and_model):
+    # Arrange
+    train, test, model = iris_split_dataset_and_model
+    check = TrainTestPredictionDrift(categorical_drift_method='PSI')
+
+    # Act
+    result = check.run(train, test, model)
+
+    # Assert
+    assert_that(result.value, has_entries(
+        {'Drift score': has_entries({0: close_to(0.08, 0.01), 1: close_to(0.038, 0.01), 2: close_to(0.07, 0.01)}),
+         'Method': equal_to('Earth Mover\'s Distance')}
+    ))
+
+
+def test_binary_proba_condition_fail_threshold(drifted_data_and_model):
+    # Arrange
+    train, test, model = drifted_data_and_model
+    check = TrainTestPredictionDrift(categorical_drift_method='PSI', drift_mode='proba'
+                                     ).add_condition_drift_score_less_than()
+
+    # Act
+    result = check.run(train, test, model)
+    condition_result, *_ = check.conditions_decision(result)
+
+    # Assert
+    assert_that(result.value, has_entries(
+        {'Drift score': has_entries({0: close_to(0.23, 0.01), 1: close_to(0.23, 0.01)}),
+         'Method': equal_to('Earth Mover\'s Distance')}
+    ))
+
+    assert_that(condition_result, equal_condition_result(
+        is_pass=False,
+        name='categorical drift score < 0.15 and numerical drift score < 0.07',
+        details='Found 2 classes with model predicted probability Earth Mover\'s Distance drift' 
+                ' score above threshold: 0.07.'
+    ))
+
+
+def test_multiclass_proba_reduce_weighted(iris_split_dataset_and_model):
+    # Arrange
+    train, test, model = iris_split_dataset_and_model
+    check = TrainTestPredictionDrift(categorical_drift_method='PSI', aggregation_method='weighted')
+
+    # Act
+    result = check.run(train, test, model)
+
+    # Assert
+    # Assert
+    assert_that(result.reduce_output(), has_entries(
+        {'Weighted Drift Score': close_to(0.06, 0.01)}
+    ))
```

Comment:
Please add tests for all drift_modes and aggregations :) Will also increase coverage.